company-profiles/x-team.md

```diff
@@ -2,10 +2,29 @@
 
 ## Company blurb
 
-⚠ We don't have much information about this company yet!
+Since 2006, X-Team has provided trusted, motivated developers to companies looking to scale their teams.
 
-If you know something we don't, help us fill it in!  Here's how:
+## Company size
 
-- Read our [Contributing Guidelines](https://github.com/remoteintech/remote-jobs/blob/master/CONTRIBUTING.md)
-- Have a look at our [example company profile](https://github.com/remoteintech/remote-jobs/blob/master/company-profiles/example.md)
-- Follow the structure of the example profile and [send us a pull request with your changes to this file!](https://github.com/remoteintech/remote-jobs/edit/master/company-profiles/x-team.md)
+100-200
```

Comment:
I couldn't find the exact number on their page, so I did a look up in linkedin and found that 176 people said that work there.

Reply:
Maybe someone who works there will come along and update this. Until then, this is fine :+1: